examples/hello_world/android/app/build.gradle

```diff
@@ -56,3 +54,8 @@ android {
 flutter {
     source '../..'
 }
+
+dependencies {
+    // This is required by the Flutter Gradle Plugin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
Interesting that this is needed when it wasn't before. Do you understand why that is?

Reply:
As of now: nope, except that it makes the build pass :)

Reply:
Not needed anymore :)